testing/gdal/APKBUILD

```diff
@@ -26,7 +26,7 @@ makedepends="
 	"
 subpackages="
 	$pkgname-dev
-	py-$pkgname:py
+	py3-$pkgname:py3
```

Comment:
We usually do `provides` and `replaces` here for backwards compatibility, please see eg. py3-cffi for how this is done.